### PR TITLE
VR direct upgrade to MF

### PR DESCRIPTION
**Pull request by shuwnyuantee**
https://github.com/regentmarkets/bom-app/pull/136
https://github.com/regentmarkets/bom-web/pull/133
https://github.com/regentmarkets/bom-platform/pull/35
https://github.com/binary-com/binary-static/pull/357

### CODE REVIEW

File: src/javascript/binary/base/page.js [hunk 1]
```diff
@@ -88,7 +88,11 @@ var Client = function() {
     if(dl_info.length > 0) {
         for (var i=0;i<dl_info.length;i++) {
             if(dl_info[i].event == 'log_in') {
-                SessionStore.set('client_info', this.loginid + ':' + dl_info[i].params.bom_firstname + ':'  + dl_info[i].params.bom_lastname + ':' + dl_info[i].params.bom_email + ':' + dl_info[i].params.bom_phone);
+                SessionStore.set('client_info',
+                    this.loginid + ':' + dl_info[i].params.bom_firstname + ':'  + dl_info[i].params.bom_lastname
+                    + ':' + dl_info[i].params.bom_email + ':' + dl_info[i].params.bom_phone
```
Comment (borisyankov) [inline src/javascript/binary/base/page.js]: The plus sign must be the last sign on the line, and not the first on a new line.
Avoids an error linked to JS engine auto adding ';'

File: src/javascript/binary/base/page.js [hunk 2]
```diff
@@ -702,15 +707,14 @@ Contents.prototype = {
...
+            for (var i=0; i<loginid_array.length; i++) {
                 var loginid = loginid_array[i].id;
                 var real = loginid_array[i].real;
 
                 if (real) {
                     has_real = true;
-                    if (!check_financial) {
-                        break;
-                    }
+                    break if (!check_financial);
```
Comment (borisyankov) [inline src/javascript/binary/base/page.js]: Not sure if JS supports this type of expression. I think it doesn't.

**Comment by borisyankov** [comment]
We should look to split page.js file into several small ones.
It always gives merge conflicts and is a container for different things.

File: src/javascript/binary/base/page.js (hunk 1 above)
Comment (shuwnyuantee) [inline src/javascript/binary/base/page.js]: @borisyankov  Sorry missed you comment previously. Done, thanks

File: src/javascript/binary/base/page.js (hunk 2 above)
Comment (shuwnyuantee) [inline src/javascript/binary/base/page.js]: done, thanks

**Comment by shuwnyuantee** [comment]
@borisyankov  yes, page.js is currently too huge of ~ 900 lines. Refactor / split it will be good.

**Pull request closed without merging**